mbd-web/.env.prod

```diff
@@ -1,6 +1,6 @@
 # All variables must be prefixed with NEXT_PUBLIC_ due to static build
 NEXT_PUBLIC_DEBUG_AUTH=false
-NEXT_PUBLIC_COGNITO_REDIRECT=https://d1gfkxim9jw7hw.cloudfront.net
+NEXT_PUBLIC_COGNITO_REDIRECT=https://d2b9uyk19x41xh.cloudfront.net
```

Comment:
'eh, what'd I tell ya? Lol, I don't know why the domain would change... hopefully we can make it static or figure out a domain name